Kiss.Bff/Extern/Klantinteracties/PostKlantcontactenCustomProxy.cs

```diff
@@ -52,18 +52,17 @@ public async Task<IActionResult> PostKlantContacten([FromBody] JsonObject parsed
             return Ok(klantcontact);
         }
 
-
-
-        public async Task<JsonObject> PostKlantContact(JsonObject parsedModel)
+        [NonAction]
```

Comment:
waarom NonAction ipv priate? is dat zodat je hem rechtstreeks kan aanroepen vanuit een test?

Reply:
De methods waren al public omdat er tests voor zijn. Vanaf het moment dat ik de nieuwe parameters toevoegde, was dit attribuut om de applicatie op te kunnen starten.

Reply:
ik weet niet of dit oneigenlijk gebruik is van het NonAction attribuut. isoleren van deze methoden in een andere class zou beter zijn naar mijn mening. Maar goed buiten scope en het is nu beter dan het was.